notebooks/text_models/solutions/rnn_encoder_decoder.ipynb

```diff
@@ -4,25 +4,33 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.        """Replicates the original preprocess_sentence function using tf.strings."""
</pre><p> </p><p>Remove this description?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/719/discussion/#comment-2431166171'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>